arm-sql/2015-05-01-preview/swagger/virtualNetworkRules.json

```diff
@@ -0,0 +1,336 @@
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/virtualNetworkRules/{vnetFirewallRuleName}": {
+      "get": {
+        "tags": [
+          "VnetFirewallRules"
+        ],
+        "description": "Gets a virtual network rule.",
+        "operationId": "VnetFirewallRules_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "vnetFirewallRuleName",
+            "in": "path",
+            "description": "The name of the virtual network rule.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved a specified virtual network rule.",
+            "schema": {
+              "$ref": "#/definitions/VnetFirewallRule"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-examples": {
+          "Gets a virtual network rule": {
+            "$ref": "../examples/VirtualNetworkRulesGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "VnetFirewallRules"
+        ],
+        "description": "Creates or updates an existing virtual network rule.",
+        "operationId": "VnetFirewallRules_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "vnetFirewallRuleName",
+            "in": "path",
+            "description": "The name of the virtual network rule.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The requested vnetFirewall Resource state.",
+            "required": true,
+            "schema": {
```

Comment:
nit: Just want to point out that the VnetFirewallRule model has no required properties which means it could theoretically be an empty object, please verify that the service is fine with this.